pkg/plugins/hybrid/v1alpha/testdata/memcached-operator/Dockerfile

```diff
@@ -10,24 +10,26 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.goå
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
-WORKDIR /
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
```

Comment:
I think this is mostly for consistency, since the helm-operator is currently based on UBI and the hybrid operator is intended to allow the user to expand on an existing helm-operator. We could change it out for distroless but we'd want to verify that doesn't cause any issues for existing users (I don't think it should)

Reply:
Cool, makes total sense. :+1: